challenges/objects-arrays.js

```diff
@@ -52,7 +75,17 @@ const graduates = [
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities);
+
+
+for (i = 0; i < graduates.length; i++) {
+  graduates[i].university;    
+  universities.push(graduates[i].university);
+  universities.sort();
```

Comment:
are you sure you want to be sorting after EVERY loop? Moving it outside the For Loop Scope would save you a set on every iteration of the loop.